webpack.config.js

```diff
@@ -0,0 +1,134 @@
+const path = require("path");
+const webpack = require("webpack");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const ESLintPlugin = require("eslint-webpack-plugin");
+const { CleanWebpackPlugin } = require("clean-webpack-plugin");
+const CopyWebpackPlugin = require("copy-webpack-plugin");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+
+const isDev = process.env.npm_lifecycle_event === "start";
+
+const base = require("./tsconfig.paths.json");
+
+function parseString(stringToParse) {
+  const regex = new RegExp("^([a-z\\/@]+)(\\/\\*)+$");
+  return stringToParse.match(regex)[1];
+}
+
+function getPaths() {
+  const {
+    compilerOptions: { paths },
+  } = base;
+  const resolvePath = (pathName) => path.resolve(__dirname, pathName);
+  return Object.keys(paths).reduce((result, key) => {
```

Comment:
```suggestion
  return Object.keys(paths).reduce((result, key) => ({
  ...result,
  [parseString(key)]: resolvePath(parseString(paths[key][0]))
  }), {})
```

Так можно сделать, потому что мы знаем, что исходный массив небольшой и здесь создание объекта/деструктуризация выполняется быстро. Если бы был массив из 10к/100к то создание на каждой итерации объекта было бы сложно.